frontend/src/pages/HomePage/HomePage.js

```diff
@@ -1,136 +1,21 @@
-                  <Placeholder.Line length='short' />
-                </Placeholder.Paragraph>
-              </Placeholder>
-            </Segment>
-          </Grid.Column>
-
-          <Grid.Column>
-            <Segment raised>
-              <Placeholder>
-                <Placeholder.Header image>
-                  <Placeholder.Line />
-                  <Placeholder.Line />
-                </Placeholder.Header>
-                <Placeholder.Paragraph>
-                  <Placeholder.Line length='medium' />
-                  <Placeholder.Line length='short' />
-                </Placeholder.Paragraph>
-              </Placeholder>
-            </Segment>
-          </Grid.Column>
-
-          <Grid.Column>
-            <Segment raised>
-              <Placeholder>
-                <Placeholder.Header image>
-                  <Placeholder.Line />
-                  <Placeholder.Line />
-                </Placeholder.Header>
-                <Placeholder.Paragraph>
-                  <Placeholder.Line length='medium' />
-                  <Placeholder.Line length='short' />
-                </Placeholder.Paragraph>
-              </Placeholder>
-            </Segment>
-          </Grid.Column>
-        </Grid>
-      </div>
-
-      <div className='placholders-temp'>
-        <Grid columns={3} stackable>
-          <Grid.Column>
-            <Segment raised>
-              <Placeholder>
-                <Placeholder.Header image>
-                  <Placeholder.Line />
-                  <Placeholder.Line />
-                </Placeholder.Header>
-                <Placeholder.Paragraph>
-                  <Placeholder.Line length='medium' />
-                  <Placeholder.Line length='short' />
-                </Placeholder.Paragraph>
-              </Placeholder>
-            </Segment>
-          </Grid.Column>
-
-          <Grid.Column>
-            <Segment raised>
-              <Placeholder>
-                <Placeholder.Header image>
-                  <Placeholder.Line />
-                  <Placeholder.Line />
-                </Placeholder.Header>
-                <Placeholder.Paragraph>
-                  <Placeholder.Line length='medium' />
-                  <Placeholder.Line length='short' />
-                </Placeholder.Paragraph>
-              </Placeholder>
-            </Segment>
-          </Grid.Column>
-
-          <Grid.Column>
-            <Segment raised>
-              <Placeholder>
-                <Placeholder.Header image>
-                  <Placeholder.Line />
-                  <Placeholder.Line />
-                </Placeholder.Header>
-                <Placeholder.Paragraph>
-                  <Placeholder.Line length='medium' />
-                  <Placeholder.Line length='short' />
-                </Placeholder.Paragraph>
-              </Placeholder>
-            </Segment>
-          </Grid.Column>
-        </Grid>
+        <HomePageList posts={data} loading={loading} />
```

Comment:
```suggestion
    { loading ?  <Loader />  : <HomePageList posts={data} /> }    
```
loading state is useless in HomePage, it will cause cluttering in the future when you have additional props